frontend/src/features/properties/list/PropertyListView.tsx

```diff
@@ -283,12 +291,12 @@ const PropertyListView: React.FC = () => {
           </div>
           <TooltipWrapper toolTipId="export-to-excel" toolTip="Export to Excel">
             <FileIcon>
-              <FaFileExcel size={36} onClick={() => fetch('excel')} />
+              <FaFileExcel size={36} title="Export to Excel" onClick={() => fetch('excel')} />
```

Comment:
so it can be found in tests

Reply:
This will actually cause some text to appear when you hover over it. If this is desired that's fine but I also see we have a tooltip wrapper here which makes it a little redundant. Another option is to add a `data-testid` attribute. After that you can query by it. See more [here](https://testing-library.com/docs/dom-testing-library/api-queries/#bytestid). 

I see that we do this on the project page though, so maybe this is something we could clean up there as well. See pic below for reference.

![image](https://user-images.githubusercontent.com/15724124/101959824-f5e03700-3bba-11eb-8cb0-c2a43bcac9e3.png)

Reply:
We don't want both, and generally the tooltip is preferred over the `title` approach.  However the `title` approach may work better inside of input dropdowns.

Reply:
I did notice that as well, I'll look into that Jonny